efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -210,3 +217,410 @@ def get_config(self):
+    self.act_type = act_type
+    self.use_tpu = use_tpu
+    self.data_format = data_format
+    self.use_dc = survival_prob and is_training
+
+    self.conv_ops = []
+    self.bn_act_ops = []
+
+    for i in range(self.repeats):
+      # If using SeparableConv2D
+      if self.separable_conv:
+        self.conv_ops.append(tf.keras.layers.SeparableConv2D(
+            filters=self.num_filters,
+            depth_multiplier=1,
+            pointwise_initializer=tf.initializers.VarianceScaling(),
+            depthwise_initializer=tf.initializers.VarianceScaling(),
+            data_format=self.data_format,
+            kernel_size=3,
+            activation=None,
+            bias_initializer=tf.zeros_initializer(),
+            padding='same',
+            name=f'{self.name}/class-%d' % i))
+      # If using Conv2d
+      else:
+        self.conv_ops.append(tf.keras.layers.Conv2D(
+            filters=self.num_filters,
+            kernel_initializer=tf.random_normal_initializer(stddev=0.01),
+            data_format=self.data_format,
+            kernel_size=3,
+            activation=None,
+            bias_initializer=tf.zeros_initializer(),
+            padding='same',
+            name=f'{self.name}/class-%d' % i))
+
+      # Level only apply here so it's maybe better inside (no need to use tf.AUTO_REUSE anymore)
+      bn_act_ops_per_level = {}
+      for level in range(self.min_level, self.max_level + 1):
+        bn_act_ops_per_level[level] = keras.utils_keras.BatchNormAct(
+            self.is_training,
+            act_type=self.act_type,
+            init_zero=False,
+            use_tpu=self.use_tpu,
+            data_format=self.data_format,
+            name='class-%d-bn-%d' % (i, level),
+            parent_name=self.name)
+      self.bn_act_ops.append(bn_act_ops_per_level)
+
+    if self.use_dc:
+      self.dc = keras.utils_keras.DropConnect(self.survival_prob)
+
+    if self.separable_conv:
+      self.classes = tf.keras.layers.SeparableConv2D(
+          filters=self.num_classes * self.num_anchors,
+          depth_multiplier=1,
+          pointwise_initializer=tf.initializers.VarianceScaling(),
+          depthwise_initializer=tf.initializers.VarianceScaling(),
+          data_format=self.data_format,
+          kernel_size=3,
+          activation=None,
+          bias_initializer=tf.constant_initializer(
+              -np.math.log((1 - 0.01) / 0.01)),
+          padding='same',
+          name=f'{self.name}/class-predict')
+
+    else:
+      self.classes = tf.keras.layers.Conv2D(
+          filters=self.num_classes * self.num_anchors,
+          kernel_initializer=tf.random_normal_initializer(stddev=0.01),
+          data_format=self.data_format,
+          kernel_size=3,
+          activation=None,
+          bias_initializer=tf.constant_initializer(
+              -np.math.log((1 - 0.01) / 0.01)),
+          padding='same',
+          name=f'{self.name}/class-predict')
+
+  def call(self, inputs, **kwargs):
```

Comment:
Hmm, our comment docstring format is like this:

def function(x, y):
  """One-line description of the function.

  Args:
     x: what is x
     y: what is y

  Returns:
     xxx
"